test/python/circuit/test_parameters.py

```diff
@@ -1178,7 +1178,30 @@ def test_compare_to_value_when_bound(self):
 
         x = Parameter("x")
         bound_expr = x.bind({x: 2.3})
+
         self.assertEqual(bound_expr, 2.3)
+        self.assertTrue(bound_expr < 3.0)
+        self.assertTrue(bound_expr > 1.0)
+        self.assertEqual(abs(bound_expr), 2.3)
+
+    def test_raise_if_compare_not_supported(self):
+        """Verify raises if compare to object."""
+        x = Parameter("x")
+        y = object
+
+        with self.assertRaisesRegex(TypeError, "not supported"):
+            x.__lt__(y)
```

Comment:
It'd be clearer to write `x < y` if that's what you mean; it's good to test the exact paths you expect to be used.